src/socks-to-rtc/socks-to-rtc.ts

```diff
@@ -163,7 +163,10 @@ module SocksToRtc {
       this.onceStopped_ = this.onceStopping_.then(this.stopResources_);
       this.onceStopped_.then(this.dispatchEvent_.bind(this, 'stopped'));
 
-      return onceReady;
+      var rejectOnStopping = new Promise((F, R) => {
```

Comment:
It's definitely fine for now and anyway my fault for missing in the original code review.

:+1: